pkg/ocm/client.go

```diff
@@ -118,7 +118,12 @@ func (b *ClientBuilder) Build() (result *Client, err error) {
 	// values in the configuration, so that default values won't be overridden:
 	builder := sdk.NewConnectionBuilder()
 	builder.Logger(logger)
-	builder.Agent(info.UserAgent + "/" + info.Version + " " + sdk.DefaultAgent)
+
+	userAgent := info.DefaultUserAgent
+	if b.cfg.UserAgent != "" {
+		userAgent = b.cfg.UserAgent
+	}
+	builder.Agent(userAgent + "/" + info.Version + " " + sdk.DefaultAgent)
```

Comment:
In your custom agent will you also be supplying capa version/release?

Reply:
Initially this wasn't part of the requirements, but your comment made think more of it and I think it makes sense to be added as well. Pushed a new commit. Thanks!